conf/scalardb-custom-values.yaml

```diff
@@ -81,36 +84,41 @@ scalardb:
   grafanaDashboard:
     enabled: false
 
-  resources: {}
+  resources:
```

Comment:
I updated the `resources.limits` and `resources.requests` based on the limitation of the commercial license.